node/core/approval-voting/src/approval_db/v1/mod.rs

```diff
@@ -76,6 +79,13 @@ impl Backend for DbBackend {
 		load_all_blocks(&*self.inner, &self.config)
 	}
 
+	fn load_all_candidates(&self) -> SubsystemResult<Vec<persisted_entries::CandidateEntry>> {
+		Ok(load_all_candidates(&*self.inner, &self.config)?
+			.into_iter()
+			.map(Into::into)
+			.collect())
```

Comment:
nit: We could avoid collecting here, and instead return the iterator instead.

Reply:
As the entire Backend trait returns `Vec` instead of `Iterator`, I think we should leave this for a later PR.